package.json

```diff
@@ -62,7 +66,8 @@
     "docs:prepare": "./docs/bin/build-preview-site.sh",
     "docs:start": "cd docs/_preview && bundle exec jekyll serve --no-w --i",
     "mocha": "node packages/build/bin/run-mocha \"packages/*/dist/__tests__/**/*.js\" \"extensions/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
-    "posttest": "npm run lint"
+    "posttest": "npm run lint",
+    "docker:setup": "node ./docker.setup.js"
```

Comment:
Since the Docker setup is specific to Cloudant, can we move this npm script to `acceptance/repository-cloudant/package.json` please?

Reply:
Yes :smile: I mentioned in the description above that once the script is released in `connector-cloudant`, the path will be changed to `"docker:setup": "node node_modules/loopback-connector-cloudant/docker.setup.js" `

Reply:
it will be loaded from dependency `loopback-connector-cloudant`.

Reply:
@jannyHou I moved the whole script to the acceptance/repo-cloudant. But I am okay with either way~